### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Markdown Toolbar_Keyboard: Keyboard focus is not visible on Toolbar Buttons.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/azuredatastudio#10653]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#MAS2.4.7;#DesktopApp;#SQL Azure Data Studio;#FTP;#Keyboard;#Benchmark;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider (system setup)
Commit: 8288360
Date: 2020-05-29T05:54:49.731Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References: MAS2.4.7

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Navigate to File --> New Notebook.
4. Navigate upto + Cell button in notebook, hit enter & select markdown cell.
5.Navigate using keyboard tab key and check whether the keyboard focus is visible or not on cell Toolbar Buttons.

**Actual:**
Keyboard focus is not visible on cell toolbar buttons.

**Expected:**
Keyboard focus should be visible on cell toolbar buttons.

**User Impact:**
User will get confuse as the focus is not clearly visible on interactive controls.

**Attachment link for Reference:**

[10653_A11y_AzureDataStudioStayHealthy_Notebook_Markdown Toolbar_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/4707532/10653_A11y_AzureDataStudioStayHealthy_Notebook_Markdown.Toolbar_Keyboard.zip)

**Comment by Rama8345** [comment]
Verified the issue at Azure Data Studio. the issue is not reproducible. So, closing the bug. Please find attachment for reference.
**Github Tags:**#Closed;
![10653_Closed](https://user-images.githubusercontent.com/115216656/221554247-e254b67e-9daf-4ac0-84c7-a9f90f6e8f3c.gif)
